concepticondata/conceptlists/Luangthongkum-2019-341.tsv

```diff
@@ -58,15 +58,15 @@ Luangthongkum-2019-341-56	56	classifier for plants, e.g. bamboo, rice, banana et
 Luangthongkum-2019-341-57	57	classfier for round objects		
 Luangthongkum-2019-341-58	58	classifier for long objects		
 Luangthongkum-2019-341-59	59	classifier for four-legged animals		
-Luangthongkum-2019-341-60	60	close (one’s eyes)	2063	CLOSE (EYES)
+Luangthongkum-2019-341-60	60	close (one’s eyes)	2211	CLOSE THE EYES
```

Comment:
[CLOSE THE EYES](https://concepticon.clld.org/parameters/2211) seems to be discontinued, so the mapping should remain